PetAdoption-iOS/PetAdoption-iOS/Views/TransparentNavBar.swift

```diff
@@ -20,9 +20,10 @@ class TransparentNavBar: UINavigationBar
         // Change the font and size of nav bar text
         if let navBarFont = UIFont(name: "OpenSans-Regular", size: 18.0)
         {
-            let navBarAttributesDictionary: [NSAttributedStringKey: AnyObject]? = [
-                NSAttributedStringKey.foregroundColor.rawValue: UIColor.white,
-                NSAttributedStringKey.font.rawValue: navBarFont
+            let navBarAttributesDictionary: [NSAttributedStringKey: AnyObject]? =
```

Comment:
the ? at the end isn't required since you are making the array.

Reply:
Also since you have declared what the array will be you can omit the NSAttributedStringKey from each key.